docs/java-rest/high-level/security/get-user-privileges.asciidoc

```diff
@@ -0,0 +1,46 @@
+--
+:api: get-user-privileges
+:request: GetUserPrivilegesRequest
+:response: GetUserPrivilegesResponse
+--
+
+[id="{upid}-{api}"]
+=== Get User Privileges API
+
+include::../execution-no-req.asciidoc[]
```

Comment:
Thanks for this. We could/should change to include it in `get-certificates.asciidoc` and `authenticate.asciidoc` too.

Reply:
Yes - I plan to do so (but that doesn't have to get done by FF, so I haven't tackled it yet)